Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -1313,6 +1365,34 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
     }
 }
 
+public class EphemeralMessageEditor {
+    private unowned let channelController: ChatChannelController
+    private var messageId: MessageId?
```

Comment:
This cold potentially be a dictionary, so that we could support having multiple live ephemeral messages